### ISSUE TITLE
9GB ISO Ubuntu error

### ISSUE THREAD

**Issue by Boshen97** [pbatard/rufus#1998]
﻿

Checklist
---------
- [x] I looked at https://github.com/pbatard/rufus/wiki/FAQ to see if my question has already been answered.
- [x] I performed a search in the issue tracker for similar issues using keywords relevant to my problem, such as the error message I got from the log.
- [x] I clicked the 'Log' button or pressed <kbd>Ctrl</kbd>-<kbd>L</kbd> in Rufus, and copy/pasted the log into the line that says `<FULL LOG>` below.
- [x] The log I am copying is the FULL log, starting with the line `Rufus version: 3.18.1877` - I have NOT removed any part of it.

Additionally (if applicable):
- [x] I ran a bad blocks check, by clicking _Show advanced format options_ then _Check device for bad blocks_, and confirmed that my USB is not defective.
- [x] I also tried one or more of the following:
  - [x] Using a different USB drive.
  - [x] Plugging the USB into a different port.
  - [ ] Running Rufus on a different computer.
- [x] If using an image, I clicked on the `(✓)` button to compute the MD5, SHA1 and SHA256 checksums, which are therefore present in the log I copied. I confirmed, by performing an internet search, that these values match the ones from the official image.

Issue description
-----------------
I made an Ubuntu system image iso file (Ubuntu system version is 18.04), this ISO file is about 9 G, I found that I couldn’t write the system disk normally, and the burned system disk was only 150MB, and I can't use it either. I don't think it's written normally, but the Ubuntu and Windows system image files can be written normally, so I want to know if my image file can be written normally? , if possible, what should I do to burn it in, please guide me.

Log
---
```
Rufus x86 v3.18.1877
Windows version: Windows 10 Pro, 64-bit (Build 19044.1826)
Syslinux versions: 4.07/2013-07-25, 6.04/pre1
Grub versions: 0.4.6a, 2.06
System locale ID: 0x0804 (zh-CN)
Will use default UI locale 0x0804
SetLGP: Successfully set NoDriveTypeAutorun policy to 0x0000009E
Localization set to 'zh-CN'
Found USB 3.0 device 'SanDisk Ultra USB 3.0 USB Device' (0781:5591)
Using 'autorun.inf' label for drive F: 'Prometheus'
1 device found
Disk type: Removable, Disk size: 128 GB, Sector size: 512 bytes
Cylinders: 14961, Tracks per cylinder: 255, Sectors per track: 63
Partition type: MBR, NB Partitions: 1
Disk ID: 0x001ECED7
Drive has a Syslinux Master Boot Record
Partition 1:
  Type: NTFS (0x07)
  Detected File System: NTFS
  Size: 114.6 GB (123059830784 bytes)
  Start Sector: 2048, Boot: Yes
Scanning image...
ISO analysis:
  Image is an ISO9660 image
  Will use '/isolinux/isolinux.cfg' for Syslinux
  Detected Syslinux version: 6.03/20171017 (from '/isolinux/isolinux.bin')
Disk image analysis:
  Image does not have a Boot Marker
ISO label: 'sblive'
  Size: 8.6 GB (Projected)
  Has a >4GB file
  Uses: Syslinux/Isolinux v6.03
  Uses: EFI
Using image: sblive.iso (8.6 GB)

Format operation started
Requesting disk access...
Will use 'F:' as volume mountpoint
Opened \\.\PhysicalDrive1 for exclusive write access
Analyzing existing boot records...
Drive has a Syslinux Master Boot Record
Clearing MBR/PBR/GPT structures...
Erasing 128 sectors
Initializing disk...
Partitioning (GPT)...
● Creating Main Data Partition (offset: 1048576, size: 114.6 GB)
● Creating UEFI:NTFS Partition (offset: 123059785216, size: 1 MB)
Writing UEFI:NTFS data...
Waiting for logical drive to reappear...
Formatting to NTFS (using IFS)
Using cluster size: 4096 bytes
Quick format was selected
Creating file system...
Format completed.
Writing Master Boot Record...
Using Rufus protective MBR
Writing protective message SBR
Found volume \\?\Volume{13877549-0c94-11ed-8944-f0d5bff6c22c}\
Disabling file indexing...
Successfully remounted \\?\Volume{13877549-0c94-11ed-8944-f0d5bff6c22c}\ as F:
Extracting files...
Image is an ISO9660 image
This image will be extracted using Joliet extensions (if present)
Extracting: F:\.disk\info (61 bytes)
Extracting: F:\EFI\BOOT\BOOTx64.EFI (1.2 MB)
Extracting: F:\EFI\BOOT\grubx64.efi (1 MB)
Extracting: F:\boot\grub\efi.img (2.3 MB)
Extracting: F:\boot\grub\font.pf2 (4.9 KB)
Extracting: F:\boot\grub\grub.cfg (812 bytes)
Extracting: F:\boot\grub\loopback.cfg (755 bytes)
Extracting: F:\boot\grub\splash.png (86.0 KB)
Extracting: F:\boot\grub\theme.cfg (597 bytes)
Extracting: F:\boot\grub\x86_64-efi\acpi.mod (16.1 KB)
Extracting: F:\boot\grub\x86_64-efi\adler32.mod (2.0 KB)
Extracting: F:\boot\grub\x86_64-efi\ahci.mod (22.1 KB)
Extracting: F:\boot\grub\x86_64-efi\all_video.mod (1 KB)
Extracting: F:\boot\grub\x86_64-efi\aout.mod (1.6 KB)
Extracting: F:\boot\grub\x86_64-efi\appleldr.mod (5.2 KB)
Extracting: F:\boot\grub\x86_64-efi\archelp.mod (4.5 KB)
Extracting: F:\boot\grub\x86_64-efi\at_keyboard.mod (7.4 KB)
Extracting: F:\boot\grub\x86_64-efi\ata.mod (8.8 KB)
Extracting: F:\boot\grub\x86_64-efi\backtrace.mod (2.6 KB)
Extracting: F:\boot\grub\x86_64-efi\bfs.mod (9.8 KB)
Extracting: F:\boot\grub\x86_64-efi\bitmap.mod (3.2 KB)
Extracting: F:\boot\grub\x86_64-efi\bitmap_scale.mod (5.2 KB)
Extracting: F:\boot\grub\x86_64-efi\blocklist.mod (3.2 KB)
Extracting: F:\boot\grub\x86_64-efi\boot.mod (3.3 KB)
Extracting: F:\boot\grub\x86_64-efi\bsd.mod (48.1 KB)
Extracting: F:\boot\grub\x86_64-efi\btrfs.mod (19.2 KB)
Extracting: F:\boot\grub\x86_64-efi\bufio.mod (2.9 KB)
Extracting: F:\boot\grub\x86_64-efi\cat.mod (4.2 KB)
Extracting: F:\boot\grub\x86_64-efi\cbfs.mod (5.4 KB)
Extracting: F:\boot\grub\x86_64-efi\cbls.mod (6.2 KB)
Extracting: F:\boot\grub\x86_64-efi\cbmemc.mod (3.9 KB)
Extracting: F:\boot\grub\x86_64-efi\cbtable.mod (1.7 KB)
Extracting: F:\boot\grub\x86_64-efi\cbtime.mod (4.1 KB)
Extracting: F:\boot\grub\x86_64-efi\chain.mod (8.6 KB)
Extracting: F:\boot\grub\x86_64-efi\cmdline_cat_test.mod (4.7 KB)
Extracting: F:\boot\grub\x86_64-efi\cmp.mod (2.9 KB)
Extracting: F:\boot\grub\x86_64-efi\command.lst (3.5 KB)
Extracting: F:\boot\grub\x86_64-efi\cpio.mod (4.3 KB)
Extracting: F:\boot\grub\x86_64-efi\cpio_be.mod (4.3 KB)
Extracting: F:\boot\grub\x86_64-efi\cpuid.mod (2.6 KB)
Extracting: F:\boot\grub\x86_64-efi\crc64.mod (2.2 KB)
Extracting: F:\boot\grub\x86_64-efi\crypto.lst (936 bytes)
Extracting: F:\boot\grub\x86_64-efi\crypto.mod (6.9 KB)
Extracting: F:\boot\grub\x86_64-efi\cryptodisk.mod (15.6 KB)
Extracting: F:\boot\grub\x86_64-efi\cs5536.mod (3.9 KB)
Extracting: F:\boot\grub\x86_64-efi\date.mod (3.3 KB)
Extracting: F:\boot\grub\x86_64-efi\datehook.mod (3 KB)
Extracting: F:\boot\grub\x86_64-efi\datetime.mod (2.0 KB)
Extracting: F:\boot\grub\x86_64-efi\disk.mod (3.1 KB)
Extracting: F:\boot\grub\x86_64-efi\diskfilter.mod (13.5 KB)
Extracting: F:\boot\grub\x86_64-efi\div_test.mod (5.7 KB)
Extracting: F:\boot\grub\x86_64-efi\dm_nv.mod (2.7 KB)
Extracting: F:\boot\grub\x86_64-efi\echo.mod (3.1 KB)
Extracting: F:\boot\grub\x86_64-efi\efi_gop.mod (13.4 KB)
Extracting: F:\boot\grub\x86_64-efi\efi_uga.mod (7.4 KB)
Extracting: F:\boot\grub\x86_64-efi\efifwsetup.mod (2.3 KB)
Extracting: F:\boot\grub\x86_64-efi\efinet.mod (6.9 KB)
Extracting: F:\boot\grub\x86_64-efi\ehci.mod (24.5 KB)
Extracting: F:\boot\grub\x86_64-efi\elf.mod (7.2 KB)
Extracting: F:\boot\grub\x86_64-efi\eval.mod (2.2 KB)
Extracting: F:\boot\grub\x86_64-efi\exfat.mod (8.3 KB)
Extracting: F:\boot\grub\x86_64-efi\exfctest.mod (2.3 KB)
Extracting: F:\boot\grub\x86_64-efi\ext2.mod (8.4 KB)
Extracting: F:\boot\grub\x86_64-efi\fat.mod (8.5 KB)
Extracting: F:\boot\grub\x86_64-efi\file.mod (23.9 KB)
Extracting: F:\boot\grub\x86_64-efi\fixvideo.mod (3 KB)
Extracting: F:\boot\grub\x86_64-efi\font.mod (17.5 KB)
Extracting: F:\boot\grub\x86_64-efi\fs.lst (214 bytes)
Extracting: F:\boot\grub\x86_64-efi\gcry_arcfour.mod (2.5 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_blowfish.mod (9.1 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_camellia.mod (31.2 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_cast5.mod (14.4 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_crc.mod (3.9 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_des.mod (16.6 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_dsa.mod (3.5 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_idea.mod (4.1 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_md4.mod (4.2 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_md5.mod (4.7 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_rfc2268.mod (3.3 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_rijndael.mod (20.5 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_rmd160.mod (7.9 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_rsa.mod (3.5 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_seed.mod (15.8 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_serpent.mod (16.7 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_sha1.mod (7.7 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_sha256.mod (5.4 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_sha512.mod (6.4 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_tiger.mod (13.4 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_twofish.mod (33.3 KB)
Extracting: F:\boot\grub\x86_64-efi\gcry_whirlpool.mod (22.4 KB)
Extracting: F:\boot\grub\x86_64-efi\geli.mod (8.9 KB)
Extracting: F:\boot\grub\x86_64-efi\gettext.mod (8.5 KB)
Extracting: F:\boot\grub\x86_64-efi\gfxmenu.mod (58.6 KB)
Extracting: F:\boot\grub\x86_64-efi\gfxterm.mod (17.6 KB)
Extracting: F:\boot\grub\x86_64-efi\gfxterm_background.mod (4.7 KB)
Extracting: F:\boot\grub\x86_64-efi\gfxterm_menu.mod (7.7 KB)
Extracting: F:\boot\grub\x86_64-efi\gptsync.mod (5 KB)
Extracting: F:\boot\grub\x86_64-efi\grub.cfg (219 bytes)
Extracting: F:\boot\grub\x86_64-efi\gzio.mod (11.2 KB)
Extracting: F:\boot\grub\x86_64-efi\halt.mod (6.7 KB)
Extracting: F:\boot\grub\x86_64-efi\hashsum.mod (8.5 KB)
Extracting: F:\boot\grub\x86_64-efi\hdparm.mod (11.1 KB)
Extracting: F:\boot\grub\x86_64-efi\help.mod (4.0 KB)
Extracting: F:\boot\grub\x86_64-efi\hexdump.mod (4.5 KB)
Extracting: F:\boot\grub\x86_64-efi\hfs.mod (10.3 KB)
Extracting: F:\boot\grub\x86_64-efi\hfsplus.mod (10.5 KB)
Extracting: F:\boot\grub\x86_64-efi\hfspluscomp.mod (4.3 KB)
Extracting: F:\boot\grub\x86_64-efi\http.mod (8.8 KB)
Extracting: F:\boot\grub\x86_64-efi\iorw.mod (4.5 KB)
Extracting: F:\boot\grub\x86_64-efi\jfs.mod (8.8 KB)
Extracting: F:\boot\grub\x86_64-efi\jpeg.mod (8.6 KB)
Extracting: F:\boot\grub\x86_64-efi\keylayouts.mod (6.3 KB)
Extracting: F:\boot\grub\x86_64-efi\keystatus.mod (3.1 KB)
Extracting: F:\boot\grub\x86_64-efi\ldm.mod (8.2 KB)
Extracting: F:\boot\grub\x86_64-efi\legacy_password_test.mod (15.6 KB)
Extracting: F:\boot\grub\x86_64-efi\legacycfg.mod (43.5 KB)
Extracting: F:\boot\grub\x86_64-efi\linux.mod (21.3 KB)
Extracting: F:\boot\grub\x86_64-efi\linux16.mod (12.6 KB)
Extracting: F:\boot\grub\x86_64-efi\linuxefi.mod (10.2 KB)
Extracting: F:\boot\grub\x86_64-efi\loadbios.mod (4.8 KB)
Extracting: F:\boot\grub\x86_64-efi\loadenv.mod (9.3 KB)
Extracting: F:\boot\grub\x86_64-efi\loopback.mod (4.8 KB)
Extracting: F:\boot\grub\x86_64-efi\ls.mod (6.6 KB)
Extracting: F:\boot\grub\x86_64-efi\lsacpi.mod (7.2 KB)
Extracting: F:\boot\grub\x86_64-efi\lsefi.mod (5.3 KB)
Extracting: F:\boot\grub\x86_64-efi\lsefimmap.mod (3.4 KB)
Extracting: F:\boot\grub\x86_64-efi\lsefisystab.mod (3.5 KB)
Extracting: F:\boot\grub\x86_64-efi\lsmmap.mod (2.9 KB)
Extracting: F:\boot\grub\x86_64-efi\lspci.mod (7.2 KB)
Extracting: F:\boot\grub\x86_64-efi\lssal.mod (3.9 KB)
Extracting: F:\boot\grub\x86_64-efi\luks.mod (9.3 KB)
Extracting: F:\boot\grub\x86_64-efi\lvm.mod (9.6 KB)
Extracting: F:\boot\grub\x86_64-efi\lzopio.mod (12.0 KB)
Extracting: F:\boot\grub\x86_64-efi\macbless.mod (4.9 KB)
Extracting: F:\boot\grub\x86_64-efi\macho.mod (10.6 KB)
Extracting: F:\boot\grub\x86_64-efi\mdraid09.mod (2.8 KB)
Extracting: F:\boot\grub\x86_64-efi\mdraid09_be.mod (2.8 KB)
Extracting: F:\boot\grub\x86_64-efi\mdraid1x.mod (2.7 KB)
Extracting: F:\boot\grub\x86_64-efi\memrw.mod (4.5 KB)
Extracting: F:\boot\grub\x86_64-efi\minicmd.mod (5.5 KB)
Extracting: F:\boot\grub\x86_64-efi\minix2.mod (5.5 KB)
Extracting: F:\boot\grub\x86_64-efi\minix2_be.mod (5.6 KB)
Extracting: F:\boot\grub\x86_64-efi\minix3.mod (5.5 KB)
Extracting: F:\boot\grub\x86_64-efi\minix3_be.mod (5.5 KB)
Extracting: F:\boot\grub\x86_64-efi\minix_be.mod (5.5 KB)
Extracting: F:\boot\grub\x86_64-efi\mmap.mod (9.5 KB)
Extracting: F:\boot\grub\x86_64-efi\moddep.lst (4.7 KB)
Extracting: F:\boot\grub\x86_64-efi\morse.mod (3.4 KB)
Extracting: F:\boot\grub\x86_64-efi\mpi.mod (42.7 KB)
Extracting: F:\boot\grub\x86_64-efi\msdospart.mod (3.7 KB)
Extracting: F:\boot\grub\x86_64-efi\multiboot.mod (18.4 KB)
Extracting: F:\boot\grub\x86_64-efi\multiboot2.mod (20.2 KB)
Extracting: F:\boot\grub\x86_64-efi\nativedisk.mod (6.5 KB)
Extracting: F:\boot\grub\x86_64-efi\net.mod (72 KB)
Extracting: F:\boot\grub\x86_64-efi\newc.mod (4.5 KB)
Extracting: F:\boot\grub\x86_64-efi\ntfs.mod (15 KB)
Extracting: F:\boot\grub\x86_64-efi\ntfscomp.mod (5.7 KB)
Extracting: F:\boot\grub\x86_64-efi\odc.mod (4.3 KB)
Extracting: F:\boot\grub\x86_64-efi\offsetio.mod (2.3 KB)
Extracting: F:\boot\grub\x86_64-efi\ohci.mod (15.8 KB)
Extracting: F:\boot\grub\x86_64-efi\part_acorn.mod (2.3 KB)
Extracting: F:\boot\grub\x86_64-efi\part_amiga.mod (2.6 KB)
Extracting: F:\boot\grub\x86_64-efi\part_apple.mod (3 KB)
Extracting: F:\boot\grub\x86_64-efi\part_bsd.mod (4.2 KB)
Extracting: F:\boot\grub\x86_64-efi\part_dfly.mod (2.7 KB)
Extracting: F:\boot\grub\x86_64-efi\part_dvh.mod (2.2 KB)
Extracting: F:\boot\grub\x86_64-efi\part_gpt.mod (3.3 KB)
Extracting: F:\boot\grub\x86_64-efi\part_msdos.mod (3 KB)
Extracting: F:\boot\grub\x86_64-efi\part_plan.mod (2.6 KB)
Extracting: F:\boot\grub\x86_64-efi\part_sun.mod (2.3 KB)
Extracting: F:\boot\grub\x86_64-efi\part_sunpc.mod (2.5 KB)
Extracting: F:\boot\grub\x86_64-efi\partmap.lst (111 bytes)
Extracting: F:\boot\grub\x86_64-efi\parttool.lst (17 bytes)
Extracting: F:\boot\grub\x86_64-efi\parttool.mod (7.3 KB)
Extracting: F:\boot\grub\x86_64-efi\password.mod (3.0 KB)
Extracting: F:\boot\grub\x86_64-efi\password_pbkdf2.mod (4.4 KB)
Extracting: F:\boot\grub\x86_64-efi\pata.mod (7.2 KB)
Extracting: F:\boot\grub\x86_64-efi\pbkdf2.mod (2.1 KB)
Extracting: F:\boot\grub\x86_64-efi\pbkdf2_test.mod (3.5 KB)
Extracting: F:\boot\grub\x86_64-efi\pcidump.mod (3.6 KB)
Extracting: F:\boot\grub\x86_64-efi\play.mod (3.9 KB)
Extracting: F:\boot\grub\x86_64-efi\png.mod (10.1 KB)
Extracting: F:\boot\grub\x86_64-efi\priority_queue.mod (2.2 KB)
Extracting: F:\boot\grub\x86_64-efi\probe.mod (4.2 KB)
Extracting: F:\boot\grub\x86_64-efi\procfs.mod (3.5 KB)
Extracting: F:\boot\grub\x86_64-efi\progress.mod (3.0 KB)
Extracting: F:\boot\grub\x86_64-efi\raid5rec.mod (2.1 KB)
Extracting: F:\boot\grub\x86_64-efi\raid6rec.mod (3.3 KB)
Extracting: F:\boot\grub\x86_64-efi\read.mod (2.3 KB)
Extracting: F:\boot\grub\x86_64-efi\reboot.mod (1.9 KB)
Extracting: F:\boot\grub\x86_64-efi\regexp.mod (75 KB)
Extracting: F:\boot\grub\x86_64-efi\reiserfs.mod (13.8 KB)
Extracting: F:\boot\grub\x86_64-efi\relocator.mod (25.3 KB)
Extracting: F:\boot\grub\x86_64-efi\romfs.mod (5.7 KB)
Extracting: F:\boot\grub\x86_64-efi\scsi.mod (7.2 KB)
Extracting: F:\boot\grub\x86_64-efi\serial.mod (14.3 KB)
Extracting: F:\boot\grub\x86_64-efi\setjmp.mod (1 KB)
Extracting: F:\boot\grub\x86_64-efi\setjmp_test.mod (2.7 KB)
Extracting: F:\boot\grub\x86_64-efi\setpci.mod (8.8 KB)
Extracting: F:\boot\grub\x86_64-efi\signature_test.mod (8.6 KB)
Extracting: F:\boot\grub\x86_64-efi\sleep.mod (3.6 KB)
Extracting: F:\boot\grub\x86_64-efi\sleep_test.mod (3.5 KB)
Extracting: F:\boot\grub\x86_64-efi\spkmodem.mod (3.2 KB)
Extracting: F:\boot\grub\x86_64-efi\squash4.mod (9.4 KB)
Extracting: F:\boot\grub\x86_64-efi\syslinuxcfg.mod (28.9 KB)
Extracting: F:\boot\grub\x86_64-efi\terminal.lst (162 bytes)
Extracting: F:\boot\grub\x86_64-efi\terminal.mod (6.7 KB)
Extracting: F:\boot\grub\x86_64-efi\terminfo.mod (19.2 KB)
Extracting: F:\boot\grub\x86_64-efi\test.mod (7.6 KB)
Extracting: F:\boot\grub\x86_64-efi\test_blockarg.mod (2.1 KB)
Extracting: F:\boot\grub\x86_64-efi\testload.mod (3.8 KB)
Extracting: F:\boot\grub\x86_64-efi\testspeed.mod (3.5 KB)
Extracting: F:\boot\grub\x86_64-efi\tftp.mod (8.1 KB)
Extracting: F:\boot\grub\x86_64-efi\tga.mod (6.5 KB)
Extracting: F:\boot\grub\x86_64-efi\time.mod (2.4 KB)
Extracting: F:\boot\grub\x86_64-efi\tr.mod (3.7 KB)
Extracting: F:\boot\grub\x86_64-efi\trig.mod (2 KB)
Extracting: F:\boot\grub\x86_64-efi\true.mod (1.9 KB)
Extracting: F:\boot\grub\x86_64-efi\udf.mod (11.1 KB)
Extracting: F:\boot\grub\x86_64-efi\ufs1.mod (7.7 KB)
Extracting: F:\boot\grub\x86_64-efi\ufs1_be.mod (7.9 KB)
Extracting: F:\boot\grub\x86_64-efi\ufs2.mod (7.7 KB)
Extracting: F:\boot\grub\x86_64-efi\uhci.mod (10.1 KB)
Extracting: F:\boot\grub\x86_64-efi\usb.mod (15.6 KB)
Extracting: F:\boot\grub\x86_64-efi\usb_keyboard.mod (5.8 KB)
Extracting: F:\boot\grub\x86_64-efi\usbms.mod (11 KB)
Extracting: F:\boot\grub\x86_64-efi\usbserial_common.mod (3 KB)
Extracting: F:\boot\grub\x86_64-efi\usbserial_ftdi.mod (3.5 KB)
Extracting: F:\boot\grub\x86_64-efi\usbserial_pl2303.mod (3.8 KB)
Extracting: F:\boot\grub\x86_64-efi\usbserial_usbdebug.mod (2.4 KB)
Extracting: F:\boot\grub\x86_64-efi\usbtest.mod (5.5 KB)
Extracting: F:\boot\grub\x86_64-efi\verify.mod (19.2 KB)
Extracting: F:\boot\grub\x86_64-efi\video.lst (41 bytes)
Extracting: F:\boot\grub\x86_64-efi\video.mod (9.0 KB)
Extracting: F:\boot\grub\x86_64-efi\video_bochs.mod (9.1 KB)
Extracting: F:\boot\grub\x86_64-efi\video_cirrus.mod (9.5 KB)
Extracting: F:\boot\grub\x86_64-efi\video_colors.mod (9.9 KB)
Extracting: F:\boot\grub\x86_64-efi\video_fb.mod (28.6 KB)
Extracting: F:\boot\grub\x86_64-efi\videoinfo.mod (5.7 KB)
Extracting: F:\boot\grub\x86_64-efi\videotest.mod (5.4 KB)
Extracting: F:\boot\grub\x86_64-efi\videotest_checksum.mod (3.8 KB)
Extracting: F:\boot\grub\x86_64-efi\xfs.mod (10.4 KB)
Extracting: F:\boot\grub\x86_64-efi\xnu.mod (40.9 KB)
Extracting: F:\boot\grub\x86_64-efi\xnu_uuid.mod (3.3 KB)
Extracting: F:\boot\grub\x86_64-efi\xnu_uuid_test.mod (3.3 KB)
Extracting: F:\boot\grub\x86_64-efi\xzio.mod (20 KB)
Extracting: F:\boot\grub\x86_64-efi\zfscrypt.mod (8.4 KB)
libcdio: Non consecutive multiextent file parts for ''
Extracting: F:\casper\initrd.gz (65.4 MB)
Extracting: F:\casper\vmlinuz (10.5 MB)
Extracting: F:\isolinux\boot.cat (2 KB)
Extracting: F:\isolinux\isolinux.bin (38 KB)
Extracting: F:\isolinux\isolinux.cfg (784 bytes)
Extracting: F:\isolinux\ldlinux.c32 (113.1 KB)
Extracting: F:\isolinux\libcom32.c32 (176.3 KB)
Extracting: F:\isolinux\libutil.c32 (22.5 KB)
Extracting: F:\isolinux\splash.png (86.0 KB)
Extracting: F:\isolinux\vesamenu.c32 (26.1 KB)
Created: F:\syslinux.cfg → /isolinux/isolinux.cfg
Finalizing, please wait...
Created: F:autorun.inf
Created: F:autorun.ico
NTFS Fixup (Checkdisk)...
Volume label is Prometheus.

Stage 1: Examining basic file system structure ...
  512 file records processed.
File verification completed.
 Phase duration (File record verification): 24.40 milliseconds.
  0 large file records processed.
 Phase duration (Orphan file record recovery): 9.98 milliseconds.
  0 bad file records processed.
 Phase duration (Bad file record checking): 8.52 milliseconds.

Stage 2: Examining file name linkage ...
  550 index entries processed.
Index verification completed.
 Phase duration (Index verification): 63.79 milliseconds.
 Phase duration (Orphan reconnection): 0.21 milliseconds.
 Phase duration (Orphan recovery to lost and found): 0.03 milliseconds.
  0 reparse records processed.
  0 reparse records processed.
 Phase duration (Reparse point and Object ID verification): 13.60 milliseconds.

Stage 3: Examining security descriptors ...
Security descriptor verification completed.
 Phase duration (Security descriptor verification): 12.08 milliseconds.
  19 data files processed.
 Phase duration (Data attribute verification): 8.18 milliseconds.

Windows has scanned the file system and found no problems.
No further action is required.

 120174547 KB total disk space.
    107076 KB in 267 files.
       148 KB in 21 indexes.
     70131 KB in use by the system.
     65536 KB occupied by the log file.
 119997192 KB available on disk.
      4096 bytes in each allocation unit.
  30043636 total allocation units on disk.
  29999298 allocation units available on disk.
Total duration: 154.03 milliseconds (154 ms).
NTFS Fixup completed.

Found USB 3.0 device 'SanDisk Ultra USB 3.0 USB Device' (0781:5591)
Using 'autorun.inf' label for drive F: 'Prometheus'
1 device found
Disk type: Removable, Disk size: 128 GB, Sector size: 512 bytes
Cylinders: 14961, Tracks per cylinder: 255, Sectors per track: 63
Partition type: GPT, NB Partitions: 2
Disk GUID: {2AA427EC-CCCE-4678-8253-41D351C7B792}
Max parts: 128, Start Offset: 17408, Usable = 123060845056 bytes
Partition 1:
  Type: Microsoft Basic Data Partition
  Name: 'Main Data Partition'
  Detected File System: NTFS
  ID: {97A56AB1-01AE-4D5B-AE59-825A999E9483}
  Size: 114.6 GB (123058736640 bytes)
  Start Sector: 2048, Attributes: 0x0000000000000000
Partition 2 (UEFI:NTFS):
  Type: Microsoft Basic Data Partition
  Name: 'UEFI:NTFS'
  Detected File System: FAT12
  ID: {6CE5C7B2-D52A-4AEC-9802-B90F3910DACD}
  Size: 1 MB (1048576 bytes)
  Start Sector: 240351143, Attributes: 0x9000000000000000
```

**Comment by JonnyTech** [comment]
> libcdio: Non consecutive multiextent file parts for ''

This looks like a `libcdio` bug linked to Joliet extensions (see https://www.mail-archive.com/libcdio-devel@gnu.org/msg01927.html). @pbatard will be able to give a more comprehensive answer (and eventually a solution!), but for now can you describe the process that you used to create the image?

**Comment by pbatard** [comment]
Yes, this looks like the issue that was fixed in Rufus 3.19 with ae377ae8cafea7a5454d1b1b0b0966bbf1f4b6f4.

You are running an __old__ version of Rufus, so please upgrade to Rufus 3.19 and you should find that your issue is fixed.
For the record, this is why you should enable the check for updates, because bugs like this get fixed all the time in newer releases, and by not having updates enabled, you are not being alerted to new versions and are running into issues that have been fixed.

As I believe upgrading to Rufus 3.19 should fix your problem, I will close this issue.
